### PR TITLE
[FEATURE] #100187 - Introduce php-intl based date&time formatting

### PR DESCRIPTION
**Pull request by brotkrueml**
Resolves: https://github.com/TYPO3-Documentation/Changelog-To-Doc/issues/393
Releases: main

### CODE REVIEW

File: Documentation/Functions/Stdwrap.rst [hunk 1]
```diff
@@ -717,6 +722,87 @@ strftime
...
+    *   `FULLTIME`, for example: `3:00:00 AM Pacific Daylight Time`
+    *   `LONGDATE`, for example: `March 17, 2023`
+    *   `LONGTIME`, for example: `3:00:00 AM PDT`
+    *   `MEDIUMDATE`, for example: `Mar 17, 2023`
+    *   `MEDIUMTIME`, for example: `3:00:00 AM`
+    *   `SHORTDATE`, for example: `3/17/23`
+    *   `SHORTTIME`, for example: `3:00 AM`
+
+    ..  note::
+        You can specify an own pattern to suit your requirements, for example:
```
Comment (linawolf) [inline Documentation/Functions/Stdwrap.rst:762]:
```suggestion
        You can specify your own pattern to suit your requirements, for example:
```

**Review by linawolf — APPROVED** [review]

File: Documentation/Functions/Stdwrap.rst (hunk 1 above)
Comment (brotkrueml) [reply Documentation/Functions/Stdwrap.rst:762]: Double "your" - would like to avoid that.

Comment (linawolf) [reply Documentation/Functions/Stdwrap.rst:762]: Specify your own pattern ...?

Comment (brotkrueml) [reply Documentation/Functions/Stdwrap.rst:762]: specify **your** own pattern to suit **your** requirements

Comment (linawolf) [reply Documentation/Functions/Stdwrap.rst:762]: Well, whatever

**Pull request merged**